standard/links/commons/links.lua

```diff
@@ -50,6 +50,7 @@ local PREFIXES = {
 	booyah = {'https://booyah.live/'},
 	bracket = {''},
 	breakingpoint = {match = 'https://www.breakingpoint.gg/match/'},
+	chesscom = {'https://www.chess.com/'},
```

Comment:
Need a match prefix too to be enabled for match summaries (for which you added data below)

Reply:
```suggestion
	chesscom = {
		'https://www.chess.com/',
		match = 'https://www.chess.com/',
	},
```

Reply:
Preferred if it's a sublink though